### PR TITLE
uqmi: add plmn set functionality for netifd proto handler

### PR DESCRIPTION
**Pull request by feckert**
uqmi has the possibility to allow the modem to start a regsitration
process only to this specified plmn

Signed-off-by: Florian Eckert <Eckert.Florian@googlemail.com>

### CODE REVIEW

**Comment by malaakso** [comment]
I think this would be a good addition. Please rebase and fix the typo in "Settting PLMN to $plmn".

File: package/network/utils/uqmi/files/lib/netifd/proto/qmi.sh [hunk 1]
```diff
@@ -76,6 +77,25 @@ proto_qmi_setup() {
...
+	[ -n "$plmn" ] && {
+		local mcc mnc
+		if [ "$plmn" = 0 ]; then
+			mcc=0
+			mnc=0
+			echo "Setting PLMN to auto"
+		else
+			mcc=${plmn:0:3}
+			mnc=${plmn:3}
+			echo "Settting PLMN to $plmn"
```
Comment (malaakso) [inline package/network/utils/uqmi/files/lib/netifd/proto/qmi.sh]: Extra 't' here.

**Comment by feckert** [comment]
Thanks i have overlooked your hint about the typo. Sorry :-)

**Comment by malaakso** [comment]
@blogic can you take it from here?

**Comment by blogic** [comment]
will do tonight, busy with other stuff just now

**Comment by blogic** [comment]
merged, Thanks !

**Pull request closed without merging**